addons/website_slides/static/src/xml/website_slides_sidebar.xml

```diff
@@ -4,17 +4,20 @@
     <t t-name="website.slides.sidebar.done.button">
         <div class="o_wslides_sidebar_done_button"
             t-att-data-id="slideId"
+            t-att-data-slide-icon="slideIcon"
             t-att-data-completed="slideCompleted"
             t-att-data-can-self-mark-completed="canSelfMarkCompleted"
             t-att-data-can-self-mark-uncompleted="canSelfMarkUncompleted"
             t-att-data-is-member="isMember"> <!-- The template is only used for members -->
             <button class="o_wslides_button_complete btn btn-sm" t-if="slideCompleted and canSelfMarkUncompleted or !slideCompleted and canSelfMarkCompleted">
                 <i t-if="slideCompleted" class="o_wslides_slide_completed fa fa-check-circle fa-fw text-success fa-lg" t-att-data-slide-id="slideId" title="Mark as not done"/>
-                <i t-if="!slideCompleted" class="fa fa-circle-thin fa-fw fa-lg" t-att-data-slide-id="slideId" title="Mark as done"/>
+                <i t-elif="slideIcon" t-attf-class="fa #{slideIcon} fa-fw fa-lg" t-att-data-slide-id="slideId" title="Mark as done"/>
+                <i t-else="" class="fa fa-circle-thin fa-fw fa-lg" t-att-data-slide-id="slideId" title="Mark as done"/>
```

Comment:
Can't we merge both, like ``t-attf-class="fa #[slideIcon or 'fa-circle-thin'} fa-fw fa-lg"`` ?